content/foundations/color.mdx

```diff
@@ -236,6 +237,49 @@ Not all colors pair well with each other. There are combinations of backgrounds
   src="https://user-images.githubusercontent.com/378023/156498652-b1121c86-a670-4a92-9a8d-9db3c7e0ed9a.png"
 />
 
+## Accessibility
+When designing with colors consider the following to make your UI visually accessible.
+
+### Assure adequate color contrast
+Color contrast between text and its background must meet [required WCAG standards](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html). 
+
+The contrast requirements are:
+- 4.5:1 for normal text
+- 3:1 for large text (>24px)
+- 3:1 for UI elements and graphics
+- Decorative and disabled elements have no contrast requirement
```

Comment:
nitpick (non-blocking): This last item seems a bit off. Should we put this as part of the introduction or as a legend?

e.g.: `The contrast requirements for non-decorative and active elements are:`

Reply:
Why do you feel this is off? I would think (and ahd this myself) that when you read about contrast requirements it would make sense to include the requirements for disabled and decorative items. 

We could turn it around to fit more into the style:
`No contrast requirement for decorative and disabled elements`

Reply:
The three first items in the list are requirements to meet the criteria, while the last point is not a requirement, but an observation. 

> We could turn it around to fit more into the style:
> `No contrast requirement for decorative and disabled elements`
> 

That sounds a bit better 👍

Reply:
Or you could say its a "non-requirement". As it is specifically listed in the specs.

Will update the sentence order for now.